docs/components/CreateNftSection.tsx

```diff
@@ -28,112 +29,129 @@ export const CreateNftSection = () => {
 
   const initialValues: FormData = { name: "", image: null };
 
+  useEffect(() => {
```

Comment:
Is this diff supposed to be part of the PR?

Reply:
Oops! Yeah I accidentally branched from `mint-success` instead of `master`. I just merged from master so the diff should be clean now. Sorry!